### PR TITLE
7.p1

### LINKED ISSUE #18: 7.p1 Designing a Thread Pool

### PR DESCRIPTION
**Pull request by x06lan**
fix #18 
## exec

```sh
make build
make run
```

## screenshot

result

### CODE REVIEW

**Comment by huhuhu0420** [comment]
request rename the folder to `7.p1`

File: hw3/7.1/makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+all: build run 
+build:
+	gcc test.c ThreadPool.h -o test.out
```
Comment (OuChiaYun) [inline hw3/7.1/makefile]: gcc test.c ThreadPool.h -o test.out -pthread

**Comment by x06lan** [comment]
> request rename the folder to `7.p1`

done

**Comment by huhuhu0420** [comment]
7.1 folder is still there 🤔

**Comment by nonorze-VOXX** [comment]
image
look like work at origin folder, but not in new folder

**Comment by x06lan** [comment]
> 7.1 folder is still there 🤔

fixed sorry my bad

**Review by huhuhu0420 — APPROVED** [review]

**Review by OuChiaYun — APPROVED** [review]

**Pull request merged**